### DISCUSSION TITLE
Need help using gist

### DISCUSSION THREAD

**Discussion by SHADOWELITE7** [community/community#22270] [New to GitHub]
<p>So I have seen other users put images in their gists. I have tried to do a screenshot in gist through MarkDown and did the MarkDown image code and it won’t show on the gist.</p>
<p>What am I doing wrong?</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/shadowelite7">@shadowelite7</a>,</p>
<p>In order to tell you what you’re doing wrong, we need to see what you’re doing in the first place. Would you care to share a link to a Gist that you’re trying to get this to work with?</p>
<p>Thanks!</p>

**Comment by SHADOWELITE7** [comment]
<p>I will need to create a new Gist because I lost my pictures for my astronomy study guide </p>

**Comment by SHADOWELITE7** [comment]
<p>here is my practice gist</p>
<pre><code>https://gist.github.com/SHADOWELITE7/393f942d8e8c49ff00c4e15f42ea6da9
</code></pre>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/shadowelite7">@shadowelite7</a>,</p>
<p>Thanks for sharing that link. The URL that you’re trying to link the image to is only <code>2018-05-07 (3).png</code> . Where is this image located? To add images to a Gist using Markdown, the image has to be uploaded somewhere accessible.</p>
<p>Thanks!</p>

**Comment by SHADOWELITE7** [comment]
<p>The image is on my computer.<br>
Where do I upload it?</p>

**Comment by SHADOWELITE7** [comment]
<p>that may be the problem, I just don’t know how to upload images to gist</p>

**Comment by SHADOWELITE7** [comment, marked as answer]
<p>I think I found my answer</p>
<pre><code>https://gist.github.com/remarkablemark/feff40b0a522f0c41c4eff0b77ea1d47
</code></pre>

**Comment by nadiajoyce** [comment]
<p>Wonderful! I’m glad you were able to find a solution!</p>
<p>Just as a quicker, potentially easier alternative (work-around), you can also upload your image to a <em>comment</em> on your Gist itself. Then, you can copy the image link that is generated from that, and use that in the Markdown link.</p>
<p>Cheers!</p>

**Comment by kannankumar** [comment]
<p>Here’s an old Gist of mine mentioning the above method (adding images to Gist using comments):</p>
<p><a href="https://gist.github.com/kannankumar/4c613cac6d9db896062a16e1cc57d3e5" rel="nofollow noopener">https://gist.github.com/kannankumar/4c613cac6d9db896062a16e1cc57d3e5</a></p>
<p>However, the above method of adding image to comment section doesnt work now.</p>
<p><strong>Update: Github Engineering seems to be working on fixing it.</strong></p>
